specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json

```diff
@@ -533,6 +566,55 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Consumption/reservationRecommendations": {
+      "get": {
+        "tags": [
+          "ReservationRecommendations"
+        ],
+        "operationId": "reservationRecommendations_List",
+        "description": "List of recomendations for purchasing reserved instances.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/consumption/"
+        },
+        "x-ms-examples": {
+          "ReservationRecommendations": {
+            "$ref": "./examples/ReservationRecommendationsList.json"
+          },
+          "ReservationRecommendationsFilterByScopeLookBackPeriod": {
+            "$ref": "./examples/ReservationRecommendationsFilterByScopeLookBackPeriod.json"
+          }
+        },
+        "parameters": [          
```

Comment:
The ```subscriptionId``` parameter is missing which is the cause of the travis failure, see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/354117627).

Reply:
Sure! Just missed it. I have added it.